instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -89,39 +92,68 @@ func newConfig(opts []Option, role string) *config {
 		metric.WithUnit("ms"))
 	if err != nil {
 		otel.Handle(err)
+		if c.rpcDuration == nil {
+			c.rpcDuration = noopFloat64Hist{}
```

Comment:
What is the reason that we are not simply using https://pkg.go.dev/go.opentelemetry.io/otel/metric/noop#Float64Histogram here?

Reply:
Please see the PR description:

> This uses a custom no-op implementation of the histograms instead of the `noop` package to remove the question about what to do with the returned error from the `noop` instrument creation (i.e. `inst, /* how to handle? */ = noop.NewMeterProvider().Meter("").Int64Histogram("")`)

Reply:
Clarifying. Why not the following?

```suggestion
			c.rpcDuration = noop.Float64Histogram{}
```

Reply:
Hmm, probably because I forgot we exported those :laughing: 

SGTM. I'll update.

Reply:
Updated. PTAL.